src/api/include/migraphx/migraphx.hpp

```diff
@@ -1343,6 +1343,14 @@ struct onnx_options : MIGRAPHX_HANDLE_BASE(onnx_options)
     {
         call(&migraphx_onnx_options_set_limit_loop_iterations, this->get_handle_ptr(), value);
     }
+
+    /// Set absolute path for external data files
+    void set_external_data_path(const std::string& external_data_path)
```

Comment:
Need to add an option in python API as well for this. 
you can add bindings in pybind here 
https://github.com/ROCm/AMDMIGraphX/blob/eed559feb338b8972335b2aee5d48602d89583b0/src/py/migraphx_py.cpp#L480

Reply:
Done.